packages/commonwealth/client/scripts/views/components/UserTrainingSlider/UserTrainingSlider.tsx

```diff
@@ -0,0 +1,299 @@
+import useUserLoggedIn from 'client/scripts/hooks/useUserLoggedIn';
+import app from 'client/scripts/state';
+import { useFetchSelfProfileQuery } from 'client/scripts/state/api/profiles';
+import { useCommonNavigate } from 'navigation/helpers';
+import React, { useEffect } from 'react';
+import useUserOnboardingSliderMutationStore from 'state/ui/userTrainingCards';
+import CWPageLayout from 'views/components/component_kit/new_designs/CWPageLayout';
+import { ActionCard } from '../ActionCard';
+import { CWText } from '../component_kit/cw_text';
+import { CWButton } from '../component_kit/new_designs/CWButton';
+import './UserTrainingSlider.scss';
+import { UserTrainingCardTypes } from './types';
+
+const CARD_TYPES = {
```

Comment:
maybe we can move it outside of the file, to make it bit smaller and more readable? similar with the admin file